pywbem_mock/_baseprovider.py

```diff
@@ -452,11 +457,11 @@ class are returned. `None` means not supplied by client and the
           filtered in accord with the local_only, etc. arguments.
 
         Raises:
-          :class:`~pywbem.CIMError`: (CIM_ERR_NOT_FOUND) if class not found in
-          CIM repository.
 
+          :class:`~pywbem.CIMError`: (CIM_ERR_NOT_FOUND) if class not found in
```

Comment:
We are inconsistent on whether the status_code_name is in paren. Most other places it is not. Actually I like the paren.

Reply:
DONE - Added parenthesis all over the uses in pywbem_mock.